docs/updating/migration-to-cdn/vuejs-v3.md

```diff
@@ -0,0 +1,189 @@
+---
+menu-title: Vue 3+
+meta-title: Vue CKEditor 5 - migrate integration from npm to CDN | CKEditor 5 documentation
+meta-description: Migrate Vue 3+ CKEditor 5 integration from npm to CDN in a few simple steps. Learn how to install Vue 3+ CKEditor 5 integration in your project using the CDN.
+category: migrations
+order: 40
+---
+
+# Migrating Vue 3+ CKEditor&nbsp;5 integration from npm to CDN
+
+This guide will help you migrate Vue 3 CKEditor&nbsp;5 integration from an NPM-based installation to a CDN-based installation.
+
+## Prerequisites
+
+Remove the existing CKEditor&nbsp;5 packages from your project. If you are using the NPM-based installation, you can remove it by running the following command:
+
+```bash
+npm uninstall ckeditor5 ckeditor5-premium-features
+```
+
+Upgrade the CKEditor&nbsp;5 Vue 3 integration to the latest version. You can find the latest version in the {@link getting-started/integrations-cdn/vuejs-v3 Vue 3} documentation.
+
+Ensure that your testing suite uses real web browser environments for testing. If you are using `jsdom` or any other environment without a real DOM, you may need to adjust the testing suite configuration to use a real browser because CDN scripts injection might be not recognized properly in such environments.
```

Comment:
```suggestion
Ensure that your testing suite uses real web browser environments for testing. If you are using `jsdom` or any other environment without a real DOM, you may need to adjust the testing suite configuration to use a real browser because CDN script injection might not be recognized properly in such environments.
```